php-frontend/src/main/java/org/sonar/php/parser/PHPGrammar.java

```diff
@@ -1807,4 +1822,32 @@ public ExpressionTree POSTFIX_EXPRESSION() {
    * [ END ] Expression
    */
 
+  public AttributeTree ATTRIBUTE() {
+    return b.<AttributeTree>nonterminal(PHPLexicalGrammar.ATTRIBUTE).is(
+      f.attribute(
+        NAMESPACE_NAME(),
+        b.optional(b.token(PHPPunctuator.LPARENTHESIS)),
+        ARGUMENTS(),
+        b.optional(b.token(PHPPunctuator.RPARENTHESIS))
```

Comment:
I believe that's wrong. `ARGUMENTS` must be surrounded by parentheses.
I think it should be something like:
```suggestion
        b.optional(
          b.token(PHPPunctuator.LPARENTHESIS),
          ARGUMENTS(),
          b.token(PHPPunctuator.RPARENTHESIS))
```